lib/mysqlDao.js

```diff
@@ -104,6 +104,20 @@ class MysqlDao {
             });
     }
 
+    upsertBulk(insertArgs, onDupUpdateArgs) {
```

Comment:
Probably also useful to note the return expectations similar to what you documented elsewhere since this is public facing API more or less.